test/input.txt

```diff
@@ -148,6 +148,21 @@
   clear
   list
 
+##########################################################
+# test sort command
+##########################################################
+
+ # add some people
+ add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy
+ add Charlie Dickson pp/333333 e/charlie.d@nus.edu.sg a/333, gamma street t/friends t/school
+ add Dickson Ee p/444444 pe/dickson@nus.edu.sg a/444, delta street t/friends
+
+ # list to show it's not in alphabetical order
+ list
+
+ # show sorted list
+ sort
+
```

Comment:
Referring to my previous comments on why your implementation is incorrect, notice that your test-cases did not manage to catch the issue, as you had called list before sort instead. Calling list after sort will ensure that your sort is fully functional.

Reply:
My sort command's purpose is to return a sorted list of names to the user, which is why it does not persist across calls. This is stated in 'help', which I have changed. I think you missed out on it!